test/Mono.Linker.Tests.Cases.Expectations/Assertions/RemovedNameValueAttribute.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Mono.Linker.Tests.Cases.Expectations.Assertions
+{
+	/// <summary>
+	/// Verifies that literal value of the member is removed
```

Comment:
```suggestion
	/// Verifies that name of the member is removed
```

"literal value" to me reads like `_field = "literal value"`.